app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/TrackBackgrounds.kt

```diff
@@ -0,0 +1,96 @@
+package nerd.tuxmobil.fahrplan.congress.schedule
+
+import android.content.Context
+import android.text.TextUtils
+import org.xmlpull.v1.XmlPullParser
+import java.util.HashMap
+import java.util.LinkedHashMap
+import nerd.tuxmobil.fahrplan.congress.R
+
+object TrackBackgrounds {
+    fun getHashMapResource(c: Context, hashMapResId: Int): Map<String?, String?>? {
```

Comment:
- `getHashMapResource` and `buildTrackBackgroundHashMap` can be `private`.
- Please add an empty line above.
- Please rename `c` to `context`.